04_environment/index.tex

```diff
@@ -194,16 +183,16 @@ \subsection{Disaster Background}
     \label{fig:Disaster eye of the storm severity}
 \end{figure}
 
-\subsection{Future work}
+\subsubsection{Future work}
 
 There are multiple future work ideas that would be developed in the design aspect such as; a deterministic disaster which would be  designed as a straight line that accumulates during the days, once the threshold has been met then the disaster would occur. Another deterministic disaster idea would be to link the disaster’s magnitude to time. Thus, agents will have to learn from past disaster occurrences, creating a memory. Agents will start learning from past disasters and will be able to forecast the next one. Therefore, the agents will be able to forecast that if a disaster did not surface for a long period of time, then the magnitude of the next disaster will be much larger than the previous one.
 
 Another idea is that agents can invest into forecasting. Therefore, agents will have access to a history of past disasters, which would be utilised to learn and gain knowledge in order to predict the upcoming disasters.
 
 Thus, the more resources invested by the agents in forecasting, the more disaster history data will be provided. This would enable the agents, to apply machine learning techniques to such data, allowing them to acquire more accurate and precise predictions of future events and develop better risk assessments. Therefore, agents will acquire knowledge, when investing in any of their resources.
 
-\section{Common Pool}
-\subsection{Common Pool Background}
+\subsection{Common Pool}
```

Comment:
similarly.. I think common pool should also remain a section -- just like 'Foraging'